src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -1169,6 +1169,23 @@ void uct_rc_mlx5_iface_common_event_cq(uct_ib_iface_t *ib_iface,
     iface->cq[dir].cq_sn++;
 }
 
+static void uct_rc_mlx5_iface_emplace_zipped_cqes(uct_ib_mlx5_cq_t *cq,
+                                                  unsigned cqe_title_index)
+{
+    uct_ib_mlx5_cq_unzip_t *cq_unzip = &cq->cq_unzip;
+    struct mlx5_cqe64 *cqe           = uct_ib_mlx5_get_cqe(cq, cqe_title_index);
+    struct mlx5_cqe64 *unzipped_cqe  = uct_ib_mlx5_iface_cqe_unzip_start(cqe,
+                                                                         cq);
+    unsigned cqe_pos;
+
+    memcpy(cqe, unzipped_cqe, sizeof(struct mlx5_cqe64));
+    for (cqe_pos = 1; cqe_pos < cq_unzip->block_size; ++cqe_pos) {
+        cqe          = uct_ib_mlx5_get_cqe(cq, cqe_title_index + cqe_pos);
```

Comment:
can we do `cqe = uct_ib_mlx5_get_cqe(cq, cq->cq_ci)` instead?
then, remove cqe_pos, and cange loop to
`while (cqe_unzip->remaining > 0)`
to simplify the code and reuse cqe_unzip state